### PR TITLE
e2eSoft.iVCam 6.2.2

### PR DESCRIPTION
**Pull request by xxPYROxxJONESxx**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/19499)

### CODE REVIEW

File: manifests/e/e2eSoft/e2eSoftiVCam/6.2.2.0/e2eSoft.e2eSoftiVCam.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: e2eSoft.e2eSoftiVCam
```
Comment (ImJoakim) [inline manifests/e/e2eSoft/e2eSoftiVCam/6.2.2.0/e2eSoft.e2eSoftiVCam.installer.yaml:4]: Let's remove the publisher from here.

```suggestion
PackageIdentifier: e2eSoft.iVCam
```

File: manifests/e/e2eSoft/e2eSoftiVCam/6.2.2.0/e2eSoft.e2eSoftiVCam.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: e2eSoft.e2eSoftiVCam
+PackageVersion: 6.2.2.0
```
Comment (ImJoakim) [inline manifests/e/e2eSoft/e2eSoftiVCam/6.2.2.0/e2eSoft.e2eSoftiVCam.installer.yaml:5]: PackageVersion needs to match the version you find in the control panel.
image

```suggestion
PackageVersion: 6.2.2
```

File: manifests/e/e2eSoft/e2eSoftiVCam/6.2.2.0/e2eSoft.e2eSoftiVCam.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: e2eSoft.e2eSoftiVCam
```
Comment (ImJoakim) [inline manifests/e/e2eSoft/e2eSoftiVCam/6.2.2.0/e2eSoft.e2eSoftiVCam.locale.en-US.yaml:4]: Let's remove the publisher from here.

```suggestion
PackageIdentifier: e2eSoft.iVCam
```

File: manifests/e/e2eSoft/e2eSoftiVCam/6.2.2.0/e2eSoft.e2eSoftiVCam.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: e2eSoft.e2eSoftiVCam
+PackageVersion: 6.2.2.0
```
Comment (ImJoakim) [inline manifests/e/e2eSoft/e2eSoftiVCam/6.2.2.0/e2eSoft.e2eSoftiVCam.locale.en-US.yaml:5]: PackageVersion needs to match the version you find in the control panel.
image

```suggestion
PackageVersion: 6.2.2
```

File: manifests/e/e2eSoft/e2eSoftiVCam/6.2.2.0/e2eSoft.e2eSoftiVCam.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: e2eSoft.e2eSoftiVCam
+PackageVersion: 6.2.2.0
+PackageLocale: en-US
+Publisher: e2eSoft
+PackageName: e2eSoft iVCam
```
Comment (ImJoakim) [inline manifests/e/e2eSoft/e2eSoftiVCam/6.2.2.0/e2eSoft.e2eSoftiVCam.locale.en-US.yaml:8]: PackageName needs to match the name you find in the control panel.
image

```suggestion
PackageName: iVCam
```

File: manifests/e/e2eSoft/e2eSoftiVCam/6.2.2.0/e2eSoft.e2eSoftiVCam.locale.en-US.yaml [hunk 6]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: e2eSoft.e2eSoftiVCam
+PackageVersion: 6.2.2.0
+PackageLocale: en-US
+Publisher: e2eSoft
+PackageName: e2eSoft iVCam
+License: (c) e2eSoft
+ShortDescription: e2eSoft iVCam Setup
```
Comment (ImJoakim) [inline manifests/e/e2eSoft/e2eSoftiVCam/6.2.2.0/e2eSoft.e2eSoftiVCam.locale.en-US.yaml:10]: Let's add a description.

```suggestion
ShortDescription: iVCam turns your Phone/Pad into an HD webcam for Windows PC.
```

File: manifests/e/e2eSoft/e2eSoftiVCam/6.2.2.0/e2eSoft.e2eSoftiVCam.yaml [hunk 7]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: e2eSoft.e2eSoftiVCam
```
Comment (ImJoakim) [inline manifests/e/e2eSoft/e2eSoftiVCam/6.2.2.0/e2eSoft.e2eSoftiVCam.yaml:4]: Let's remove the publisher from here.

```suggestion
PackageIdentifier: e2eSoft.iVCam
```

File: manifests/e/e2eSoft/e2eSoftiVCam/6.2.2.0/e2eSoft.e2eSoftiVCam.yaml [hunk 8]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: e2eSoft.e2eSoftiVCam
+PackageVersion: 6.2.2.0
```
Comment (ImJoakim) [inline manifests/e/e2eSoft/e2eSoftiVCam/6.2.2.0/e2eSoft.e2eSoftiVCam.yaml:5]: PackageVersion needs to match the version you find in the control panel.
image

```suggestion
PackageVersion: 6.2.2
```

**Comment by ImJoakim** [comment]
Hey, please see my comments above.

Also, you need to rename the following folders and files via Github Desktop to match the suggested changes:
Folder: `e2eSoftiVCam` -> `iVCam`
Folder: `6.2.2.0` -> `6.2.2`
File: `e2eSoft.e2eSoftiVCam.installer.yaml` -> `e2eSoft.iVCam.installer.yaml`
File: `e2eSoft.e2eSoftiVCam.locale.en-US.yaml` -> `e2eSoft.iVCam.locale.en-US.yaml`
File: `e2eSoft.e2eSoftiVCam.yaml` -> `e2eSoft.iVCam.yaml`

**Comment by xxPYROxxJONESxx** [comment]
I have committed the changes you have suggested using the website - are you saying that I need to download the GitHub desktop application to do it properly? sorry, this is my first time. thanks for your help.

**Comment by ImJoakim** [comment]
> I have committed the changes you have suggested using the website - are you saying that I need to download the GitHub desktop application to do it properly? sorry, this is my first time. thanks for your help.

Hey there, that is correct. The easiest way to rename folders and files in a PR, would be via GitHub Desktop. You are not able to do that via the GitHub website.

**Comment by ImJoakim** [comment]
You can also close this PR and open up another one with the suggested changes. However, it is very useful to learn certain things regarding GitHub, especially if you plan on adding more applications to winget, among other things. So my suggestion would be to see if you are able to change it via GitHub Desktop and commit it to this PR.

**Comment by ImJoakim** [comment]
Perfect, love to see it. Now you need to wait for it to go through the 7 automated checks, followed by a manual review from an Administrator or Community Moderator.
Also, you should probably change title of the PR to match the new PackageIdentifier and PackageVersion. You can do that via the GitHub website:
image

**Comment by xxPYROxxJONESxx** [comment]
I THINK I DID IT?!? lol, thanks for the encouragement. The funny thing is, I originally had it formatted correctly just by using common sense and logic but wingetcreate spat out some errors when I tried to submit it. so I ran it again and just left it with suggestions wingetcreate gave me. I think the problem was something to do with tokens... I had created one as per the instructions of the tutorial I was watching and the removal of said token seemed to fix the issue. anyway, i think I've got the hang of it now :)

**Comment by denelon** [comment]
@xxPYROxxJONESxx if you think it's a bug in the [Windows Package Manifest Creator](https://github.com/microsoft/winget-create), feel free to 👍 the bug if it has already been reported, or create a new one.

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**